magicctapipe/scripts/lst1_magic/lst1_magic_dl2_to_dl3.py

```diff
@@ -476,8 +643,13 @@ def main():
     )
 
     parser.add_argument(
-        '--input-file-irf', '-i', dest='input_file_irf', type=str, required=True,
-        help='Path to an input IRF file.',
+        '--input-file-irf', '-i', dest='input_file_irf', type=str, required=False, default=None,
+        help='Path to an input IRF file (single).',
+    )
+
+    parser.add_argument(
+        '--input-dir-irf', dest='input_dir_irf', type=str, required=False,
+        help='Path to an input IRF directory (interpolation will be applied).',
```

Comment:
I meant here to keep the original API: depending if file or dir is passed it used a single file or does the interpolation. We could pass the dir with the same variable and then check if it is a file or a dir, but this is a bit confusing in my opinion

Reply:
Regarding this I simply removed the argument "--input-file-irf". I checked that even if the input IRF file is one, the script works by selecting the interpolation method to "nearest".